fe/fe-core/src/main/java/com/starrocks/common/util/SqlParserUtils.java

```diff
@@ -114,4 +115,20 @@ public static StatementBase parseAndAnalyzeStmt(String originStmt, ConnectContex
         statementBase.analyze(analyzer);
         return statementBase;
     }
+
+    public static StatementBase parseStmtWithNewParser(String originStmt, ConnectContext connectContext) {
```

Comment:
There is no need to add a function separately, just parse and analyze directly in the position used